src/near-rpc-provider.ts

```diff
@@ -291,7 +293,7 @@ export class NearRpcProvider extends JsonRpcProvider {
     return params
   }
 
-  async getCode(addressOrName: string | Promise<string>, blockTag: BlockTag): Promise<string> {
+  async getCode(addressOrName: string, blockTag: BlockTag): Promise<string> {
```

Comment:
let's allow people to still use `Promise` here, same as in the `base-provider`. We can do `addressOrName = await addressOrName` in our code to enable that (and similar for `blockTag`)

Reply:
Mmmh, okay!